docs/docs/models/td-full-metrics.md

```diff
@@ -0,0 +1,168 @@
+Of course, the model is not perfect, so there is a chance we will wrongly remove a video that actually
+contains an animal.
+
+To assess this tradeoff, we can use the holdout set to simulate this strategy with a range of thresholds.
+For each threshold, we compute the fraction of blank videos correctly discarded and the fraction of non-blank
+videos incorrectly discarded.
+The following figure shows the results.
+
+<img src="https://s3.amazonaws.com/drivendata-public-assets/zamba/td_full_set_recall_recall_curve.png" alt="" style="width:600px;"/>
+
+The markers indicate three levels of tolerance for losing non-blank videos. For example, if it's acceptable to
+lose 5% of non-blank videos, we can choose a threshold that removes 63% of the blank videos.
+If we can tolerate a loss of 10%, we can remove 80% of the blanks.
+And if we can tolerate a loss of 15%, we can remove 90% of the blanks.
+Above that, the percentage of lost videos increases steeply.
+
+
+### Accuracy
+
+In addition to identifying blank videos, the model also computes a probability that each species appears in each video.
+We can use these probabilities to quantify the accuracy of the model for species classification.
+Specifically, we computed:
+
+* Top-1 accuracy, which is the fraction of videos where the species with the highest predicted probability is, in fact, present.
+
+* Top-3 accuracy, which is the fraction of videos where one of the three species the model considered most likely is present.
+
+Over all videos in the holdout set, the **top-1 accuracy is 82%; the top-3 accuracy is 94%**.
+As an example, if you choose a video at random and the species with the highest predicted probability is elephant, the probability is 82% that the video contains an elephant, according to the human-generated labels.
+If the three most likely species were elephant, hippopotamus, and cattle, the probability is 94% that the video
+contains at least one of those species.
+
+These results depend in part on the species represented in a particular dataset. For example, in the small
+number of videos from Equatorial Guinea, only three species appear. For these videos, the top-1 accuracy is 97%, much
+higher than the overall accuracy.
+In the videos from Ivory Coast, 21 species are represented, so the problem is harder. For these
+videos, top-1 accuracy is 80%, a little lower than the overall accuracy.
+
+
+### Recall and precision by species
+
+One of the goals of classification is to help with retrieval, that is, efficiently finding videos containing
+a particular species. To evaluate the performance of the model for retrieval, we can use
+
+* Recall, which is the fraction of videos containing a particular species that are correctly classified, and
+
+* Precision, which is the fraction of videos the model labels with a particular species that actually contain that species.
+
+The following figure shows recall and precision for the species in the holdout set.
```

Comment:
Just to clarify (for me, but maybe also worth reflecting here), this is based on top-1 classification performance, i.e. for the prediction with the highest probability across the 22? 32? possible classes. Is that right?